docs/configuration.md

```diff
@@ -63,6 +63,7 @@ str(receptors)
 | `met_path`           | Absolute path to ARL compatible meteorological data files                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | `met_file_format`    | String detailing file naming convention for meteorological data files using a mixture of datetime and regex syntax. The formatting string accepts `grep` compatible regular expressions (`.\*.arl`), `strftime` compatible datetime strings (`%Y%m%d%H`) or any combination of the two. Datetime syntax is expanded to all unique combinations required for the receptor and simulation duration and the intersection between the requested files and files available in `met_path` is determined with `grep`, allowing partial matching and compatible regular expressions to be used to identify the relevant data. Matching does not require the full format to be specified - e.g. `\*.arl`, `%Y`, `%Y%m%d`, `%Y%m%d_d0.*.arl` would all match with a file named `20180130_d01.arl`. |
+| `n_hours_per_met_file` | Number of hours per meteorological data file. Defaults to 6                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Suggest adding guidance about how to know what value to set, since ARL met files can be a bit of a black box to users

Reply:
I haven't had any experience with ARL met files other than the HRRR archive that Lin Group keeps on CHPC. @tartanrunner25 do you have any suggestions for user guidance on determining the number of hours in an ARL met file?